lib/phony/countries/myanmar.rb

```diff
@@ -0,0 +1,18 @@
+# Myanmar: http://www.itu.int/dms_pub/itu-t/oth/02/02/T02020000920003PDFE.pdf
+# + formatting according to Google's libphonenumber library. (r680 7/30/2014)
+#
+# note that ANEE SAKHAN area, Yangon ZTE GSM, MEC Tel CDMA 800 Mhz is not
+# correctly handled by the google library
```

Comment:
I am confused here – does this mean you fixed it for Phony or that these cases are also incorrect in this PR?